static/app/views/dashboardsV2/utils.tsx

```diff
@@ -313,3 +313,7 @@ export function getDashboardsMEPQueryParams(isMEPEnabled: boolean) {
       }
     : {};
 }
+
+export function getNextEquationIndex(possibleEquations: string[]) {
+  return possibleEquations.filter(isEquation).length;
+}
```

Comment:
hmm, im not sure that this function does what is intended (or maybe the function name could use an improvement).
looks like returns the total number of equations in the array. if you have `[equation, field, field]`, it would return `1` but I think you might want `0`?

Reply:
So I guess this was supposed to be used to get the next equation index if you were trying to add one more. In the case of `[equation, field, field]`, the next equation index to use in an alias is `equation[1]`.

Maybe I should just rename it to something like `getNumEquations` and avoid trying to abstract out its intention

Reply:
Ahhh I see, makes more sense. Yeah a rename might make it easier to understand